src/analysis/normed_space/operator_norm.lean

```diff
@@ -1,193 +1,174 @@
 /-
 Copyright (c) 2019 Jan-David Salchow. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Jan-David Salchow, Sébastien Gouëzel
+Authors: Jan-David Salchow, Sébastien Gouëzel, Jean Lo
 
 The space of bounded linear maps
 
 Define the set of bounded linear maps between normed spaces and show basic facts about it. In
 particular
 
-(*) define a set L(E,F) of bounded linear maps between k normed spaces,
-(*) show that L(E,F) is a vector subspace of E → F,
-(*) define the 'operator norm' on L(E,F) and show that it induces the structure of a normed space
-    on L(E,F).
+(*) show that bounded linear maps are a vector subspace of E → F,
+(*) define the operator norm and show that it induces the structure of a normed space
+    on bounded linear maps.
 -/
 import algebra.module
 import analysis.normed_space.bounded_linear_maps
+import topology.metric_space.lipschitz
 
 variables {k : Type*}
-variables {E : Type*} {F : Type*}
+variables {E : Type*} {F : Type*} {G : Type*}
 
+variables [normed_field k]
+variables [normed_space k E] [normed_space k F] [normed_space k G]
 
--- Define the subspace of bounded linear maps.
-section bounded_linear_maps
+noncomputable theory
 set_option class.instance_max_depth 50
 
-
-variables [hnfk : normed_field k] [normed_space k E] [normed_space k F]
-include hnfk
-
-def bounded_linear_maps : subspace k (E → F) :=
-{ carrier := {A : E → F | is_bounded_linear_map k A},
+def bounded_linear_map : subspace k (E → F) :=
```

Comment:
The parameters k, E and F should be explicit in `bounded_linear_map` (for instance by writing them as `(k : Type*)` and `(E : Type*)...` on lines 19 and 20, and then adding a line `variables {k E F G}` after the definition). Then you can write `bounded_linear_map α β γ` on line 36.